YACS-RCOS/yacs.n
# Fixed parser bug by removing the 'Method' column to fix parser standard

**Issue**

Closes #255 

**Test Procedure**

*Example:*
> 1. Run the file get-spring-2021.sh
> 2. Check resultant file to be sure the format is correct
> 3. Repeat steps 1&2 for get-fall-2020.csv

**Additional Info**

spring2021.csv should be good to use. Just be sure to update it often so it is current and up to date
